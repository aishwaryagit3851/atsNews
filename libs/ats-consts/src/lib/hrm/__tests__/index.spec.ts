import { EMPLOYEE_TYPES_OBJ } from '..';

describe('ATS CONSTS for - hrm', () => {
  it('Match consts to snapshot', () => {
    expect(EMPLOYEE_TYPES_OBJ).toMatchSnapshot();
  });
});
