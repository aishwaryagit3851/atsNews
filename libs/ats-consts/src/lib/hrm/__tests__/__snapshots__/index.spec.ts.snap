// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ATS CONSTS for - hrm Match consts to snapshot 1`] = `
{
  "CONTRACTOR": "CONTRACTOR",
  "EMPLOYEE": "EMPLOYEE",
}
`;
